pkg/kubelet/kubelet_pods.go

```diff
@@ -207,6 +217,29 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 	return mounts, nil
 }
 
+// translateMountPropagation transforms v1.MountPropagationMode to
+// runtimeapi.MountPropagation.
+func translateMountPropagation(enabled bool, mountMode *v1.MountPropagationMode) (runtimeapi.MountPropagation, error) {
+	if !enabled {
+		// mount propagation is disabled, use private as in the old versions
+		return runtimeapi.MountPropagation_PROPAGATION_PRIVATE, nil
+	}
+	switch {
+	case mountMode == nil:
+		// HostToContainer is the default
+		return runtimeapi.MountPropagation_PROPAGATION_HOST_TO_CONTAINER, nil
+
+	case *mountMode == v1.MountPropagationHostToContainer:
+		return runtimeapi.MountPropagation_PROPAGATION_HOST_TO_CONTAINER, nil
+
+	case *mountMode == v1.MountPropagationBidirectional:
+		return runtimeapi.MountPropagation_PROPAGATION_BIDIRECTIONAL, nil
+
+	default:
+		return 0, fmt.Errorf("Invalid MountPropagation mode: %q", mountMode)
```

Comment:
s/Invalid/invalid
Error should start with a lower-case letter

Reply:
fixed